File: SuruchiJewellersManagementFrontend/src/app/models/order/order-view-model.ts
import { OrderDetailsViewModel } from "../order-details/order-details-view-model";

export class OrderViewModel {
    id: number;
    customerName: string;
    orderNumber: string;
    vori: string;
    ana: string;
    roti: string;
    productOptionName: string;
    amount: string;
    date: string;

    orderDetailsViewModels: OrderDetailsViewModel[] = [];
}

File: Suruchi Jewellers Management Backend/SuruchiJewellersManagement/SuruchiJewellersManagement.Domain/ViewModels/Order/OrderViewModel.cs
using SuruchiJewellersManagement.Domain.ViewModels.OrderDetails;

namespace SuruchiJewellersManagement.Domain.ViewModels.Order
{
    public class OrderViewModel
    {
        public int Id { get; set; }
        public string CustomerName { get; set; }
        public string OrderNumber { get; set; }
        public string Vori { get; set; }
        public string Ana { get; set; }
        public string Roti { get; set; }
        public string ProductOptionName { get; set; }
        public string Amount { get; set; }
        public string Date { get; set; }

        public ICollection<OrderDetailsViewModel> OrderDetailsViewModels { get; set; }
    }
}